instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from "prop-types";
+
+class PostContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { likes: this.props.post.likes };

```

Comment:
Incrementing/Decrementing likes step 1:
```suggestion
    this.state = {likes: this.props.post.likes, isLiked: false};
```